examples/state_store/state_store.py

```diff
@@ -24,6 +27,12 @@
     data = d.get_state(store_name=storeName, key=key).data
     print(f"Got value: {data}")
 
+    # Transaction upsert
+    d.execute_transaction(store_name=storeName, operations=[
+        TransactionalStateOperation(key=key, data=updated_value),
```

Comment:
operation type needs to be set for each operation

Reply:
It defaults to 'upsert', do you think it should be required? I found it obvious that upsert is the most common scenario.

Reply:
default upsert is ok but I think we need to show the usage of the API with using possible options in the test and docstring by default . Default operation type behavior scenario would be additional example.